src/main/webapp/pages/projects/settings/pages/remote.html

```diff
@@ -1,69 +1,35 @@
 <!DOCTYPE html>
-<html xmlns:th="http://www.thymeleaf.org" xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
-      data-layout-decorate="~{projects/settings/_base}">
-<head>
-    <title th:text="#{project.settings.page.title.remote}">Title</title>
+<html
+  lang="en"
+  xmlns:th="http://www.thymeleaf.org"
+  xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+  data-layout-decorate="~{projects/settings/_base}"
+>
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="/dist/css/project-remote.bundle.css">
+    <title th:text="#{ProjectRemoteSettings.title}">Title</title>
         <script th:inline="javascript">
-        var PAGE = {
-            urls: {
-            'sync': /*[[@{/projects/{id}/settings/sync(id=${project.getId()})}]]*/ '/projects/1/settings/sync',
-            }
-        };
-        </script>
-</head>
-<body>
-<div layout:fragment="settings-content" th:with="status = ${project.getRemoteStatus()}">
-    <h2 class="h2-muted" th:text="#{project.settings.sync.heading}"></h2>
-    <div class="form-group">
-        <label class="control-label" th:text="#{project.settings.sync.lastSync}">_Last Sync_</label>
-        <div>
-            <span th:text="${#calendars.format(project.getRemoteStatus().getLastUpdate(), 'dd MMM yyyy hh:mma')}"></span>
-            <button id="forceSync" role="button" class="btn btn-default btn-sm"
-                    th:text="#{project.settings.sync.forceSync}">_Force Sync_
-            </button>
-        </div>
-    </div>
-    <div class="form-group connection-wrapper">
-        <label class="control-label" th:text="#{project.settings.api}">_API_</label>
-        <div>
-            <a href="#" th:href="@{/remote_api/{id}(id=${status.getApi().getId()})}"
-               th:text="${status.getApi().getLabel()}">_api_</a>
-            <span id="api-status" class="api-status js-status-wrapper">
-                <i class="fa fa-spinner fa-spin"></i>
-            </span>
+            window.PAGE = {
+                _USER: /*[[${session.CURRENT_USER_DETAILS}]]*/ {}
```

Comment:
Do you need this, I think there is a global variable for this.  Check window.TL on the page.html template

Reply:
I will take a look

Reply:
Updated in bb34052964426a9d5f05269592dbeb56afcf1e82